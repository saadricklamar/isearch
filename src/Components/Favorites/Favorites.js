import React from "react";
import { Link } from "react-router-dom";

const Favorites = () => {
  return (
    <div className="favorites">
      <header className="favorites-header">
        <Link className="go-home" to="/">
          <i class="fas fa-arrow-left"></i>
        </Link>
      </header>
      <h2>This is a Favorites page</h2>
    </div>
  );
};

export default Favorites;
