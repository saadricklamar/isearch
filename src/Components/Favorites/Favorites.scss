.favorites {
  background: url(../../Assets/Mac-About-Page.jpeg) no-repeat center center
    fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  height: 100vh;
}

.favorites-header {
  display: flex;
}

h2 {
  margin: 0 auto;
  color: white;
}

.go-home {
  color: white;
  font-size: 3rem;
  margin: 2%;
}

.go-home:hover {
  color: hotpink;
}
