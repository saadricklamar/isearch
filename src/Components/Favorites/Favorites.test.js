import React from "react";
import { shallow } from "enzyme";
import Favorites from "./Favorites";

describe("Favorites", () => {
  let wrapper = shallow(<Favorites />);
  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("renders component with correct path when back arrow is clicked", () => {
    expect(wrapper.find("Link").prop("to")).toEqual("/");
  });
});
