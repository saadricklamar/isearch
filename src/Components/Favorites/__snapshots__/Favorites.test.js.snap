// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites should match the snapshot 1`] = `ShallowWrapper {}`;
